assets/src/components/map/follower.tsx

```diff
@@ -177,53 +179,64 @@ export const usePickerContainerFollowerFn = () => {
-  // map container
-  const topLeft = new Point(445, 0)
-
-  if (points.length === 1) {
-    const targetZoom = 16
-    const innerBounds = new Bounds(topLeft, mapContainerBounds.getBottomRight())
-    // The "new center" is the offset between the two bounding boxes centers
-    const offset = innerBounds
-      .getCenter()
-      .subtract(mapContainerBounds.getCenter())
-
-    const targetPoint = map
-        // Project the target point into screenspace for the target zoom
-        .project(points[0], targetZoom)
-        // Offset the target point in screenspace to move the center of the map
-        // to apply the padding to the center
-        .subtract(offset),
-      // convert the target point to worldspace from screenspace
-      targetLatLng = map.unproject(targetPoint, targetZoom)
-
-    // Zoom/Pan center of map to offset location in worldspace
-    map.setView(targetLatLng, targetZoom)
-  } else {
-    const pointsBounds = Leaflet.latLngBounds(points)
-    map.fitBounds(pointsBounds, {
-      paddingBottomRight: [50, 20],
-      paddingTopLeft: topLeft,
-    })
+    const { width, height } = map.getContainer().getBoundingClientRect()
+    const mapContainerBounds = new Bounds([0, 0], [width, height])
+
+    // ```
+    // vpcElement.getBoundingClientRect().right - mapElement.getBoundingClientRect().left
+    //  -> 445
+    // ```
+    // Create a new inner bounds from the map bounds + "padding" to shrink the
+    // inner bounds
+    // In this case, we get the top left of the inner bounds by padding the left
+    // with the distance from the right side of the VPC to the left side of the
+    // map container
+    const topLeft = new Point(445, 0)
+
+    if (points.length === 1) {
+      const targetZoom = useTargetZoom
+        ? 16
+        : map.getZoom() < 13
+        ? 16
+        : map.getZoom()
+      const innerBounds = new Bounds(
+        topLeft,
+        mapContainerBounds.getBottomRight()
+      )
+      // The "new center" is the offset between the two bounding boxes centers
+      const offset = innerBounds
+        .getCenter()
+        .subtract(mapContainerBounds.getCenter())
+
+      const targetPoint = map
+          // Project the target point into screenspace for the target zoom
+          .project(points[0], targetZoom)
+          // Offset the target point in screenspace to move the center of the map
+          // to apply the padding to the center
+          .subtract(offset),
+        // convert the target point to worldspace from screenspace
+        targetLatLng = map.unproject(targetPoint, targetZoom)
+
+      // Zoom/Pan center of map to offset location in worldspace
+      map.setView(targetLatLng, targetZoom)
+    } else {
+      const pointsBounds = Leaflet.latLngBounds(points)
+      map.fitBounds(pointsBounds, {
+        paddingBottomRight: [50, 20],
+        paddingTopLeft: topLeft,
+      })
+    }
   }
-}
+
+export const drawerOffsetAutoCenter = drawerOffsetAutoCenter2(true)
```

Comment:
**suggestion:** how about naming this something like `fixedZoomDrawerOffsetAutoCenter`, and keeping `drawerOffsetAutoCenter` as the name for what's now called `drawerOffsetAutoCenter2`.

Reply:
💯 I definitely would reconsider a lot of this if this functionality is wanted by product/design.